module/modmanager/manager.go

```diff
@@ -769,7 +781,9 @@ func (m *module) startProcess(
 	viamHomeDir string,
 ) error {
 	var err error
-	if m.addr, err = modlib.CreateSocketAddress(filepath.Dir(parentAddr), m.cfg.Name); err != nil {
+	// append a random alpha string to the module name while creating a socket address to avoid conflicts
```

Comment:
```suggestion
	// append a random alpha string to the module name while creating a socket address to avoid conflicts
	// with old versions of the module.
```
Just a slight elaboration; let me know if that's not correct.